packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -394,10 +402,14 @@ class TxForm extends Component {
         return preValidation;
       }
 
+      if (values.gas && values.gas.toString() === '-1') {
+        return { gasPrice: 'Unable to estimate gas...' };
+      }
+
       // If the gas hasn't been calculated yet, then we don't show any errors,
       // just wait a bit more
-      if (!this.estimatedTxFee(values)) {
-        return;
+      if (!this.isEstimatedTxFee(values)) {
+        return { gasPrice: 'Estimating gas...' };
```

Comment:
Got it, this seems to be how the `validate` props of final-form works, it would only display an error once the fields has actually been modified (makes sense).
A simple workaround would be to show the error on the `amount` field, (that must users must fill). I think this is fine because the gas estimation isn't particularly related to the gas price.
```suggestion
        return { amount: 'Estimating gas...' };
```

Reply:
Agreed, yes only difference is that the popup will appear about the Amount field

Reply:
@Tbaut I've pushed a commit that adopts the workaround that you mentioned of showing the errors on the `amount` field instead